helpers/refresh_group_data.example.php

```diff
@@ -33,26 +33,27 @@
 }
 catch (Exception $e)
 {
-  print '<p><strong>Caught exception: ' . $e->getMessage() . 
+  print '<p><strong>Caught exception: ' . $e->getMessage() .
     '</strong></p>';
 }
 
 
 $cachefile = "groups_cache_dc.json";
 refresh_groups_cache($ckan,$data,$cachefile);
 
-   
+
 
 
 function refresh_groups_cache ($ckan,$data,$cachefile) {
   echo "Refreshing Group data...<br/>";
   $groups = array();
-  for ($i = 0; $i < count($data); $i++) {
-    $groups[$data[$i]] = $ckan->get_group_entity($data[$i]);
-  }
-  $groups = json_encode($groups);
-  file_put_contents($cachefile,$groups);
+	if (count($data) > 500) {
+	  for ($i = 0; $i < count($data); $i++) {
+	    $groups[$data[$i]] = $ckan->get_group_entity($data[$i]);
+	  }
```

Comment:
Mixed tabs and spaces :(

Reply:
ughhhhh... I so don't care... T_T Will fix...